docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,199 @@
+| Aanleiding     | Een consumer wil informatieobjecttypeinformatie van de informatieobjecten die voor kunnen komen bij het zaaktype zoals opgenomen in de catalogus van de organisatie opvragen.            |
+| *Toevoegen*    |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om informatieobjecttypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus het besluittype is opgenomen. |
+| - gevolg       | Een overzicht van Informatieobjecttypeinformatie zoals opgenomen in de catalogus wordt teruggegeven.             |
+| *Raadplegen details*   |      |
+| - voorwaarde   | Om details van informatieobjecttypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus het informatieobjecttype is opgenomen. Tevens moet de identficatie van het informatieobjecttype wat uitgevraagd wordt bekend zijn. |
+| - gevolg       | Informatieobjecttypeinformatie zoals opgenomen in de catalogus wordt teruggegeven.             |
+| Gegevens       | INFORMATIEOBJECTTYPE <br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/informatieobjecttype_read                                                                                |
+| Samenhang      | - |
+| Bijzonderheden | - |
+
+
+### Resource: eigenschap
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil alle eigenschappen van een zaaktype zoals opgenomen in de catalogus van de organisatie opvragen.             |
+| *Toevoegen*    |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om eigenschapinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype de eigenschap is opgenomen. |
+| - gevolg       | Een overzicht van eigenschapinformatie van het zaaktype wordt teruggegeven.            |
```

Comment:
Is toch een overzicht van eigenschappen?

Reply:
Mja, meer van eigenschap-typen zou ik zeggen. De eigenschap heeft een waarde, dat heeft de typedefnitie niet.

Reply:
Ik resolve de conversation

Reply:
Is dit consistent met andere vermeldingen van het gevolg van Raadplegen lijst?